src/MercadoPago/Resources/Preference/CategoryDescriptor.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace MercadoPago\Resources\Preference;
+
+/** CategoryDescriptor class. */
+class CategoryDescriptor
+{
+    /** Event date. */
+    public ?string $eventDate;
```

Comment:
[req] PHP segue padrão snake_case
```suggestion
    public ?string $event_date;
```